src/config/mongoose_config_validator_toml.erl

```diff
@@ -519,6 +501,10 @@ validate(V, integer, positive) -> validate_positive_integer(V);
 validate(V, integer, port) -> validate_port(V);
 validate(V, int_or_infinity, non_negative) -> validate_non_negative_integer_or_infinity(V);
 validate(V, int_or_infinity, positive) -> validate_positive_integer_or_infinity(V);
+validate(V, int_or_infinity_or_atom, positive) ->
+    validate_positive_integer_or_infinity_or_atom(V, no_buffer);
+validate(V, int_or_atom, positive) ->
+    validate_positive_integer_or_atom(V, never);
```

Comment:
We need to think about how to get rid of these strange types... I'd do something like:

- `buffer`: boolean (`disabled/enabled`)
- `buffer_max`: number/infinity

- `ack`: boolean (`disabled/enabled`)
- `ack_freq`: number

It's not mandatory, we could do it later, it just looks confusing and inconsistent as it is now.